### ISSUE TITLE
Frontend - Bindings 

### ISSUE THREAD

**Issue by ShalokShalom** [pureqml/qmlcore#160]
Hi there :)

I am puzzled - how to connect my backend code to this QML? 

Is any language fine? JavaScript only?
Do I use a binding for my language?

Thanks a lot

**Comment by whoozle** [comment]
Hello, @ShalokShalom 

We are not imposing any restriction on backend and/or frontend technology you could possibly use. 
It should work equally well with any technology, e.g. nodejs/express or django or flask or whatever you like to use on your server. 

As a friendly reminder, resulting pureqml code  is running in browser on client side. You may use any front-end libraries, or manually written js code, as long as it don't interfere with DOM or our namespace (qml). 
Normally you may extend default `index.html` template with their libraries of choice and use them in QML methods as you would do in any javascript framework.

Speaking of API and backend technology, https://swagger.io/, is a good place to start with. It's extensive and powerful tool allowing you to design API and even export them as code. They provide codegen tool, called `swagged-codegen`, you can learn more by the link:
https://swagger.io/tools/swagger-codegen/

Long story short, there's plenty of excellent technology out there, covering every aspect of possible needs. 

I'm sorry but I have to close this issue, as it's out of the scope of our project. 
Have fun!

**Comment by ShalokShalom** [comment]
Super. You could put this into your Readme. Very enlightening :hugs
